doc/source/contribution/documentation/introduction.rst

```diff
@@ -6,6 +6,19 @@ Introduction
 This documentation describes about how to write, generate and manage
 Groonga documentation.
 
+How to clone Groonga repository
+-------------------------------
+
+First, please fork Groonga repository on GitHub because
+you need to build Groonga documentation from the latest source code.
+You just access https://github.com/groonga/groonga and press
```

Comment:
```suggestion
You just need to access https://github.com/groonga/groonga and press
```

Do we need `need to` here?

Reply:
I think we had better add `need to` because we will use the same expression with the previous sentence.

Reply:
fix: 48894e4 I use the same expression with the previous sentence. Also, I removed a `just` because we will have to clone as a next step.